### ISSUE TITLE
Set up Bitbucket Pipeline for Acquia deployment

### ISSUE THREAD

**Issue by mrdavidburns** [Lullabot/drainpipe#728]
depends on: #454 

Using the `task` commands set up in #454 let's add a composer.json Drainpipe configuration that will scaffold a Bitbucket Pipeline for deploying to Acquia. Let's have it so that this happens anytime something gets merged into `main` branch.

This would match functionality that we do with [PantheonReviewApps](https://github.com/Lullabot/drainpipe/blob/acdb3b4bb5aeea90058a19af76a350b58b33b019/scaffold/github/workflows/PantheonReviewAppsDDEV.yml)

**Comment by mrdavidburns** [comment]
As discussed during Drainpipe sync (11/4/24) let's hold off on having Drainpipe do the scaffolding for Bitbucket Pipelines. Let's build those directly on the projects that need this functionality and once we have 3 or more clients that need this functionality we'll have working examples to reference to help standardize this into Drainpipe. 

Unassigning for now and setting to Blocked for reason stated above.

**Comment by mrdavidburns** [comment]
We now have 4 repos using Bitbucket. It makes sense to add this to Drainpipe.

**Comment by YesCT** [comment]
Oh, how can we link to the mirrors and keep that info with this public issue? Oh, maybe on a lsm-examples wiki page... like: https://github.com/Lullabot/lsm-examples/wiki/Implement-scripted-deployments ??? Can we try it?

@mrdavidburns You could edit https://github.com/Lullabot/lsm-examples/wiki/Implement-scripted-deployments to note the mirrors that are using this already.

(If there are projects that aren't mirror'd maybe the todo is: to ping the PMs of that project and ask them to start the mirroring process, by asking slack: mirrors?.)

**Comment by beto-aveiga** [comment]
I opened a support ticket for Bitbucket Pipelines and DDEV.
Bitbucket's support team is very responsive, so we will see what they respond.

<img width="1920" height="4810" alt="Image" src="https://github.com/user-attachments/assets/b9be5db7-fa1d-4afe-9a26-2396949eecb7" />
